mne/io/nirx/tests/test_nirx.py

```diff
@@ -605,3 +606,23 @@ def test_nirx_standard(fname, boundary_decimal):
     """Test standard operations."""
     _test_raw_reader(read_raw_nirx, fname=fname,
                      boundary_decimal=boundary_decimal)  # low fs
+
+
+@requires_testing_data
+@pytest.mark.parametrize('fname, want_order', [
+    (fname_nirx_15_0, ['S1_D1', 'S2_D2', 'S3_D3', 'S4_D4', 'S5_D5', 'S6_D6', 'S7_D7', 'S8_D8', 'S9_D9', 'S10_D10']),  # noqa: E501
+    (fname_nirx_15_2, ['S1_D1', 'S1_D10', 'S2_D1', 'S2_D2', 'S3_D2', 'S3_D3', 'S4_D3', 'S4_D4', 'S5_D4', 'S5_D5', 'S6_D5', 'S6_D6', 'S7_D6', 'S7_D7', 'S8_D7', 'S8_D8', 'S9_D8', 'S9_D9', 'S10_D9', 'S10_D10', 'S11_D11', 'S11_D12', 'S12_D12', 'S12_D13', 'S13_D13', 'S13_D14', 'S14_D14', 'S14_D15', 'S15_D15', 'S15_D16', 'S16_D11', 'S16_D16']),  # noqa: E501
+    (fname_nirx_15_2_short, ['S1_D1', 'S1_D9', 'S2_D3', 'S2_D10', 'S3_D2', 'S3_D11', 'S4_D4', 'S4_D12', 'S5_D5', 'S5_D6', 'S5_D7', 'S5_D8', 'S5_D13']),  # noqa: E501
```

Comment:
This is an example where users experience or expectations will change what they call “logical ordering”. Channels 9 and above are short channels here. These channels are all actually the same fibre optic bundle and same ADC, so one might expect them to be ordered together. An alternative expectation is that all the short channels are ordered at the end of the channel list (this is how it’s visually displayed in some vendors GUI, so a user might be accustomed to that and expect it).

Reply:
I think the end goal should be not to do any reordering (different from what the order is on disk). But as an intermediate solution we pair the fnirs channels as the first N because preprocessing steps require it

Reply:
... FYI this test was added to ensure that the pairing reordering is a noop for nirx at least. So when we remove any channel reordering, this test shouldn't need to change at all.

Reply:
Clever. Love it